docs/rebuild.sh

```diff
@@ -1,2 +1,20 @@
 #!/bin/sh
-pandoc index.md > index.html
+
+# extract '# Desription' section from root README.md
```

Comment:
Is this still going to be simple enough? In the previous-previous iteration, we had a Python script to generate the HTML. One day something went awfully wrong because it was generating a page where all the text overlapped. Someone compared the webpage to a "satanic ritual". None of us at the time had enough time to invest into fixing it, so we burned it all down and replaced it with the one line of pandoc you see now.

Reply:
Yes, it's still just all run through pandoc (see the last line), using a template to make it look nice.

The idea behind the csplit/sed/cat combo is really just to keep in sync with the root README. The current `index.md` is fairly dated.

For https://github.com/orangecms/utk-web, I have been using the same template for months now. See https://hostile.education/utk-web/about.html for the outpur.